docs/run.md

```diff
@@ -18,49 +18,68 @@ Recommended IDE is [Visual Studio Code](https://code.visualstudio.com/), which w
 
 ---
 
-### Setup Python
+### Installation
 
-You need to have `python >= 3.10`
+`python >= 3.10` and `pip` are required
 
-To achieve this easily, you can use a conda virtual environment:
-See [installation instuctions](https://docs.anaconda.com/anaconda/install/linux/).
-Then set up the environment:
+If you need to manage multiple versions of python in your system go to [Setup Python](#setup-python)
+
+Install dependencies:
 
 ```bash
-conda create -n py-ispyb python=3.10
-conda activate py-ispyb
-```
+# For development and production
+pip install -r requirements.txt
 
----
+# For development only
+pip install -r requirements-dev.txt
+```
 
-### Install requirements
+#### System requirements
 
-Install required packages
+For development purposes some packages need to be present on your system. These packages are needed for SALS (Simple Authentication and Security Layer) support, LDAP and MariaDB database development files
 
-* For Debian and derivatives:
+- For Debian and derivatives:
 
 ```bash
 sudo apt-get update && sudo apt-get install -y libldap2-dev libsasl2-dev \
 libmariadb-dev build-essential
 ```
 
-* For Fedora and derivatives (use `yum` if you don't have `dnf`):
+- For Fedora and derivatives (use `yum` if you don't have `dnf`):
 
 ```bash
 sudo dnf update && sudo dnf install -y openldap-devel mariadb-connector-c-devel \
 python3-devel
 ```
+
 For Fedora you might also need to create a text file `/usr/lib/libldap_r.so`, adding only the line `INPUT ( libldap.so )`
 
+### Setup Python
+
+Virtual environments allows to install and manage different versions of python easily.
 
-Install python dependencies:
+#### Conda virtual environment
+
+Conda is an open source package management system and environment management system. [Intallation instructions](https://docs.anaconda.com/anaconda/install/linux/)
+
+Then set up the environment:
 
 ```bash
-# For development and production
-pip install -r requirements.txt
+conda create -n py-ispyb python=3.10
+conda activate py-ispyb
+pip install -r requirements # or pip install -r requirements-dev.txt
```

Comment:
`and` rather than `or` for development, you need to install both requirements files to develop